### DISCUSSION TITLE
Prisma bug report after running "npx prisma init" command on ssh local hosting server

### DISCUSSION THREAD

**Discussion by klasikh** [prisma/prisma#21765] [Q&A]
### Bug description

I got the following error when running "npx prisma init" command :

prisma:warn Prisma doesn't know which engines to download for the Linux distro "undefined". Falling back to Prisma engines built "debian".
Please report your experience by creating an issue at https://github.com/prisma/prisma/issues so we can add your distro to the list of known supported distros.
> Downloading Prisma engines for Node-API for debian-openssl-1.0.x [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/4bc8b6e1b66cb932731fb1bdbbc550d1e010de81/debian-openssl-1.0.x/libquery_engine.so.node.gz.sha256 failed, reason: unable to get local issuer certificate

I need help, please help me.

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
The relevant error message here is `reason: unable to get local issuer certificate`. Something in your terminal is unexpected and does not respond correctly when the CLI tried to download that file.

Can you access this URL manually? https://binaries.prisma.sh/all_commits/4bc8b6e1b66cb932731fb1bdbbc550d1e010de81/debian-openssl-1.0.x/libquery_engine.so.node.gz.sha256
Can you download it in the terminal session where you are trying to use the CLI, e.g. with `wget ...` or `curl`?

**Reply by klasikh** [reply]
Yes, I can access this URL manually.

**Reply by janpio** [reply]
Can you download it in the terminal session where you are trying to use the CLI, e.g. with wget ... or curl?

**Reply by klasikh** [reply]
No, that have not work

**Reply by janpio** [reply]
Then the problem is somehow with your terminal not being able to access the internet the same as your browser.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
